internal/services/node/node.go

```diff
@@ -0,0 +1,135 @@
+package node
```

Comment:
same here, lmk if I misunderstand if we want be more strict about naming "node" vs "storage node" to avoid confusion

Reply:
Okay I think we're good on these now.